### DISCUSSION TITLE
WF3 and 4?

### DISCUSSION THREAD

**Discussion by camposdaniel21** [UiPath/corewf#134] [Q&A] [answered]
Hi! I already probably know the answer, but i need to ask (lol). Does the CoreWF support WF3 as WF4 in .Net Framework does? And its XOML?

**Comment by dmetzgar** [comment, marked as answer]
WF4 calling WF3 activities is done through the [Interop](https://docs.microsoft.com/en-us/dotnet/api/system.activities.statements.interop?view=netframework-4.8) activity. We left that out intentionally since the runtimes are so different and very few products use WF3. XOML is similar to the code-behind concept in ASP.NET. Quite different from XAML.
